src/xcode/ENA/ENA/Resources/Localization/de.lproj/usage.html

```diff
@@ -17,7 +17,7 @@
     }
     
     
-    a {
+    a[href] {

```

Comment:
Can you also update that in the en file and the `privacy-policy.html` file, please? :-)